compiler/rustc_borrowck/src/diagnostics/mod.rs

```diff
@@ -1089,7 +1089,7 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                     if is_option_or_result && maybe_reinitialized_locations_is_empty {
                         err.span_suggestion_verbose(
                             fn_call_span.shrink_to_lo(),
-                            "consider calling `.as_ref()` to borrow the type's contents",
+                            "consider calling `.as_ref()` or `.as_mut()` to borrow the type's contents",
                             "as_ref().",
                             Applicability::MachineApplicable,
```

Comment:
It looks like the user has a choice. So should we be making this `MachineApplicable`? Thinking of it again, should it be a suggestion at all or just a `span_help`?

Reply:
didn't even notice this was MachineApplicable. Yeah, it shouldn't. I could probably bump it down and make the suggestion inline, not verbose like it's rendering now